bin/test/quickstrom

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+cd elm/spec
+
+sudo rm -rf report
+
+docker stop webdriver
+
+docker run --rm -d \
+  --name webdriver \
+  --network=host \
```

Comment:
Is there a good reason to be running in host networking mode instead of `-p 4444:4444` ?

I think host networking doesn't work on Mac.

Reply:
Ah I see, it allows the second docker container to connect.

Reply:
otherwise I guess we need a `docker-compose.yml` and a dedicated network. I just copied these from the `quickstrom` documentation